common/src/claims.rs

```diff
@@ -166,6 +169,55 @@ impl Default for ScopeBuilder {
     }
 }
 
+#[derive(Clone, Copy, Debug, Default, Deserialize, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "lowercase")]
+#[cfg_attr(feature = "backend", derive(strum::Display))]
+#[cfg_attr(feature = "backend", derive(sqlx::Type))]
+#[cfg_attr(feature = "backend", sqlx(rename_all = "lowercase"))]
+#[cfg_attr(feature = "backend", strum(serialize_all = "lowercase"))]
+pub enum AccountTier {
+    #[default]
+    Basic,
+    // A basic user that is pending a payment on the backend.
+    PendingPaymentPro,
+    Pro,
+    Team,
+    Admin,
+    Deployer,
+}
+
+impl From<AccountTier> for Limits {
+    fn from(value: AccountTier) -> Self {
+        match value {
+            AccountTier::Basic | AccountTier::PendingPaymentPro | AccountTier::Deployer => {
+                Self::default()
+            }
+            AccountTier::Pro | AccountTier::Team => Self::new(MAX_PROJECTS_EXTRA),
+            AccountTier::Admin => Self::new(100),
```

Comment:
This is a bit arbitrary, since we currently ignore this limit if the claim has the `Scope::Admin` variant. Maybe this one should be default as well?

Reply:
Yip, we can make it default too